docs/modules/ROOT/pages/documentation/payara-server/public-api/openid-connect-support.adoc

```diff
@@ -336,17 +418,144 @@ Authorization Server.
 - The `getClaims()` method - Gets the User Claims that were received from the userinfo endpoint.
 - The `getProviderMetadata()` method - The OpenId Connect Provider's metadata document fetched via provider URI.
 
+[[user-information-from-the-id-token]]
+== User information from the ID Token
+
+The following property is added to get user information from the ID Token.
+
+NOTE: To integrate with Microsoft ADFS this property will be needed because by default Microsoft ADFS doesn't provide user information from the /userinfo endpoint.
+
+.Configuration Option
+|===
+| Option | MP Config property | Required | Description | Default value | Requirements
+
+| `userClaimsFromIDToken`
+| `payara.security.openid.userClaimsFromIDToken`
+| false
+| Enables to get the user information from the ID Token
+| false
+| This property should be use in case it is needed to get the user information from the ID Token.
+
+|===
+
+[[user-information-from-the-id-token-example]]
+=== Example
+
+[source, java]
+----
+@OpenIdAuthenticationDefinition(
+       providerURI = "https://sample-openid-server.com",
+       clientId = "87068hgfg5675htfv6mrucov57bknst.apps.sample.com",
+       clientSecret = "{my-secret}",
+       redirectURI = "${baseURL}/callback",
+       userClaimsFromIDToken=true
+)
+public class SecurityBean {
+
+}
+----
+
+[[disable-scope-validation]]
+== Disable scope validation
+
+To disable the scope validation it is needed to use the following property of the `OpenIdProviderMetadata`.
```

Comment:
I suggest adding more detail, for example:
```suggestion
By default, Payara OpenID Connect connector validates that a scope is reported as a supported scope by the provider. However, some providers support more scopes than they actually report as supported scopes. In order to disable the validation and allow using such scopes, it's possible to use the following property of `@OpenIdProviderMetadata`:
```

Reply:
already done